package.json

```diff
@@ -16,6 +16,7 @@
     "@babel/preset-env": "^7.18.2",
     "@babel/preset-react": "^7.17.12",
     "@loadable/component": "^5.15.2",
+    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.7",
```

Comment:
An `EXPERIMENTAL` plugin, that has over 6 million installed a week. Nice:

<img width="1182" alt="Experimental" src="https://user-images.githubusercontent.com/86275/188607848-194bd980-3fbc-4541-b966-b15654f66ee2.png">